GeoHealthCheck/plugins/probe/wfs3.py

```diff
@@ -1,25 +1,64 @@
-import requests
```

Comment:
Should we rename this file to `GeoHealthCheck/plugins/probe/oafeat.py`?

Reply:
Normally would agree. Only this is not a simple name change: like `resource_type` `OGC:WFS3`, the Probe class package string is stored in the DB (`probe_vars` table, column `probe_class`) like `GeoHealthCheck.plugins.probe.wfs3.WFS3Caps`. So this would require an Alembic DB upgrade as well. I propose to do this in separate issue/PR.

Reply:
OK. +1 to cover in another issue/PR.

Reply:
See my comment above. I agree that we should align naming for `OAFeat`. (Was not even aware of the most recent naming.) But this PR is mainly the OWSLib upgrade, causing bugs fixed for OWS Authentication (`headers` param WFS and TMS) and a need to rewrite WFS3/OAFeat Probe code. Was more involved than planned. That next PR would then be for #313.

Reply:
Though renaming OAPIF to OAFeat (is that the new name?) in the comments and some code is ok. Only the class names and Resource Type `OGC:WFS3` have DB-impact.